src/main/java/seedu/address/logic/commands/DeleteClaimCommand.java

```diff
@@ -79,7 +79,7 @@ public CommandResult execute(Model model) throws CommandException {
             model.setClient(clientToEdit, clientWithDeletedClaim);
             model.updateFilteredClientList(PREDICATE_SHOW_ALL_CLIENTS);
 
-            return new CommandResult(String.format(MESSAGE_DELETE_CLAIM_SUCCESS, Messages.format(clientToEdit),
+            return new CommandResult(String.format(MESSAGE_DELETE_CLAIM_SUCCESS, clientToEdit.getName().toString(),
```

Comment:
Just to be clear: this is not to fix the current issue right, but more to address the issue that surfaced over the weekend in our telegram chat 🙂

Reply:
yeah is to fix the tests better also because if u create before deleting, the message is gonna be different if u take the formatted client message (which includes all the info including plans and claims) to the one after.

Reply:
Makes sense. Thanks! Going to merge now